src/components/TheCollections/Week.vue

```diff
@@ -20,47 +20,71 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <template>
-	<div ng-repeat="day in days | filter:dayHasEntry(day)" class="grouped-tasks ui-droppable">
-		<h2 class="heading">
-			<!-- <text>{{ day | day  }}</text> -->
-		</h2>
-		<ol :collectionID="$route.params.collectionId"
-			class="tasks"
-			listID=""
-			type="list"
-			dnd-list="draggedTasks"
-			dnd-drop="dropAsRootTask(event, item, index)"
-			dnd-dragover="dragover(event, index)">
-			<li v-for="task in tasks"
-				:taskID="task.uri"
-				:key="task.id"
-				class="task-item ui-draggable handler"
-				ng-animate="'animate'"
-				ng-click="openDetails(task.uri,$event)"
-				ng-class="{done: task.completed}"
-				dnd-draggable="task"
-				dnd-dragstart="dragStart(event)"
-				dnd-dragend="dragEnd(event)">
-				<!-- ng-repeat="task in filtered = filteredTasks() | filter:taskAtDay(task,day) | filter:hasNoParent(task) | filter:{'completed':'false'} | orderBy:getSortOrder():settingsmodel.getById('various').sortDirection"> -->
-				<!-- dnd-effect-allowed="{{ allow(task) }}"> -->
-				<task />
-			</li>
-		</ol>
+	<div>
+		<div v-for="day in dayHasEntry(days)" :key="day" class="grouped-tasks ui-droppable">
+			<h2 class="heading">{{ day | formatDay }}</h2>
+			<ol collectionID="week"
+				class="tasks"
+				listID=""
+				type="list"
+				dnd-list="draggedTasks"
+				dnd-drop="dropAsRootTask(event, item, index)"
+				dnd-dragover="dragover(event, index)">
+				<li v-for="task in tasks"
+					:taskID="task.uri"
+					:key="task.id"
+					class="task-item ui-draggable handler"
+					ng-animate="'animate'"
+					ng-click="openDetails(task.uri,$event)"
+					ng-class="{done: task.completed}"
+					dnd-draggable="task"
+					dnd-dragstart="dragStart(event)"
+					dnd-dragend="dragEnd(event)">
+					<!-- ng-repeat="task in filtered = filteredTasks() | filter:taskAtDay(task,day) | filter:hasNoParent(task) | filter:{'completed':'false'} | orderBy:getSortOrder():settingsmodel.getById('various').sortDirection"> -->
+					<!-- dnd-effect-allowed="{{ allow(task) }}"> -->
+					<task :task="task" :tasks="tasks" />
+				</li>
+			</ol>
+		</div>
 	</div>
 </template>
 
 <script>
-import { mapState } from 'vuex'
-import Task from '../Task'
+import { mapGetters } from 'vuex'
+import TaskBody from '../Task'
 
 export default {
 	components: {
-		'task': Task
+		'task-body': TaskBody
+	},
+	filters: {
+		formatDay: function(day) {
+			var date = moment().add(day, 'day')
+			var dayString
+			if (day == 0) {
+				dayString = t('tasks', 'Today')
+			} else if (day == 1) {
+				dayString = t('tasks', 'Tomorrow')
+			} else {
+				dayString = date.format('dddd')
+			}
+			return dayString + ', ' + date.format('LL')
```

Comment:
@skjnldsv @georgehrke Do you know, if I have to configure the locale manually for the app? When I select a locale under settings->user and navigate to the Tasks app, it is shown using the correct locale. But when I reload the page, the locale is lost and the english fallback is used. However, under settings, the correct locale is still shown. Should moment.js be configured with the correct locale by the server automatically?

Reply:
@raimund-schluessler Take a look at my implementation of the dateTime with dynamic locale load and moment
https://github.com/nextcloud/contacts/blob/vue/src/components/Properties/PropertyDateTime.vue

:)

Reply:
Thanks. Is there a reason why you don't use the moment.js version built into server?

Reply:
@raimund-schluessler I can manage the updates myself and not depends on the server :)

Reply:
Also, this doesn't seem to be necessary for Calendar?!

Reply:
@raimund-schluessler what is not nexessary? :)

Also, reference of config and nice setup to not load all the locale at once: https://github.com/nextcloud/vue-components/issues/13#issuecomment-419452405

Reply:
> @raimund-schluessler what is not nexessary? :)

To configure the locale manually. At least I couldn't find it anywhere in the Calendar app code.

Reply:
Ah, no idea. :thinking:

Reply:
This https://github.com/nextcloud/tasks/pull/194/commits/9c6ca2b241d95932a1161eab520ad28ef9ed8559 seems to do the trick for now. :)